src/App.vue

```diff
@@ -13,12 +13,12 @@
         <div class="spacer"></div>
         <v-card class="nav-items has-space-right d-flex" :elevation="2" v-if="!$vuetify.breakpoint.mdAndDown">
           <v-btn to="/" :elevation="0" active-class="is-active">Home</v-btn>
-          <v-btn v-for="path of paths" :key="path.to" :to="path.to" :elevation="0" active-class="is-active">{{ path.label }}</v-btn>
+          <v-btn :id="'navbar-nav-'+path.label.replace('/','')" v-for="path of paths" :key="path.to" :to="path.to" :elevation="0" active-class="is-active">{{ path.label }}</v-btn>
```

Comment:
The `replace` method only replaces the first occurrence. You will need to use `replaceAll` instead.